### ISSUE TITLE
Histogram with specific bins

### ISSUE THREAD

**Issue by sjensen85** [d3/d3-array#77]
Hello,

I am trying to show a bar chart histogram with two different sets of data, displayed with two different series of bars. I have used this to calculate the bins and data that fits into those bins:

```
    var max = d3.max(data);
    var min = d3.min(data);
    var x = d3
      .scaleLinear()
      .domain([min, max])
      .range([0, 400]);

    var histogram = d3
      .histogram()
      .domain(x.domain())
      .thresholds(x.ticks(10));

    return histogram(data);
```

Using this, calculating the second set of data gives me separate bins. However I would like the second set of data to be in the same bins as the first. Is this possible and if so, how would I go about it? I am having a rough time finding anything for specifying the exact bins I would like to use on the second set of data.

(I didn't realize I posted this in d3-array. Let me know if I should post somewhere else.)

**Comment by mbostock** [comment]
Pass the desired bin thresholds to [*histogram*.thresholds](https://github.com/d3/d3-array/blob/master/README.md#histogram_thresholds).

**Comment by mbostock** [comment]
I don’t understand your latest comment. Can you rephrase in terms of expected vs observed behavior, ideally with live code that demonstrates the issue as an Observable notebook?
